src/main/java/com/almostreliable/summoningrituals/altar/AltarBlock.java

```diff
@@ -38,14 +38,19 @@
 
 public class AltarBlock extends Block implements EntityBlock {
 
-    static final BooleanProperty ACTIVE = BooleanProperty.create(Constants.ACTIVE);
-    private static final DirectionProperty FACING = BlockStateProperties.HORIZONTAL_FACING;
-    private static final VoxelShape SHAPE = Stream.of(
+    public static final BooleanProperty ACTIVE = BooleanProperty.create(Constants.ACTIVE);
+    public static final DirectionProperty FACING = BlockStateProperties.HORIZONTAL_FACING;
+    public static final VoxelShape SHAPE = Stream.of(
```

Comment:
I figured in case someone would want to extend the AltarBlock class in the future it'd make a lot of things easier for them to implement whatever changes they'd need that depends on these states and the methods that I've made protected. But, I suppose that can be done when that use case arises.